synapse/tests/test_axon.py

```diff
@@ -322,6 +407,104 @@ def test_axon_host_clone(self):
             axonbyts = b''.join(_byts for _byts in axonc2.iterblob(blob))
             self.eq(axonbyts, cv)
 
+            # We can get stats from an Axon proxy object
+            iden1 = axon0.alloc(8)
+            axon0.chunk(iden1, b'hehe')
+
+            ekeys = [
```

Comment:
I think these assertions (duplicated above) could be moved into a separate function like the one you created for general Cortex assertions.